packages/@webex/plugin-meetings/test/unit/spec/meetings/index.js

```diff
@@ -637,6 +637,29 @@ describe('plugin-meetings', () => {
 
         const FAKE_USE_RANDOM_DELAY = true;
         const correlationId = 'my-correlationId';
+        const callStateForMetrics = {
+          correlationId: 'my-correlationId2',
+          joinTrigger: 'my-join-trigger',
+          loginType: 'my-login-type',
+        };
+
+        it('should call setCallStateForMetrics on any pre-existing meeting', async () => {
+          const fakeMeeting = {setCallStateForMetrics: sinon.mock()};
+          webex.meetings.meetingCollection.getByKey = sinon.stub().returns(fakeMeeting);
+          await webex.meetings.create(
+            test1,
+            test2,
+            FAKE_USE_RANDOM_DELAY,
+            {},
+            correlationId,
+            true,
+            callStateForMetrics
```

Comment:
this is an example where we provide 2 different correlationd ids, which one is the right/correct one?

Reply:
yeah, I went for correlationId because it was more explicitly set, but I wouldn't expect anyone to set both. There's a test further down that tests it's over written explicity.